content/zh-cn/docs/concepts/security/pod-security-admission.md

```diff
@@ -158,7 +157,6 @@ Kubernetes 定义了一组{{< glossary_tooltip term_id="label" text="标签" >}}
 要采取什么样的动作。
 
 <!--
-{{< table caption="Pod Security Admission modes" >}}
```

Comment:
This change breaks Netlify build.

Reply:
Got it. The break was caused by the `{{< /table >}}` in the commented lines. Looks like these shortcode are parsed even if they are commented out.